src/@utils/aquarius.ts

```diff
@@ -40,7 +46,7 @@ export function generateBaseQuery(
         filter: [
           ...(baseQueryParams.filters || []),
           getFilterTerm('chainId', baseQueryParams.chainIds),
-          getFilterTerm('_index', 'aquarius'),
+          // getFilterTerm('_index', 'aquarius'),
```

Comment:
why was this commented?

Reply:
think filter was not working at that time, uncommented it and all fine 😃